src/ParcelRegistry.Api.Legacy/Parcel/ParcelController.cs

```diff
@@ -87,7 +101,13 @@ public ParcelController(IMediator mediator)
         [SwaggerResponseExample(StatusCodes.Status500InternalServerError, typeof(InternalServerErrorResponseExamples))]
         public async Task<IActionResult> Count(CancellationToken cancellationToken = default)
         {
-            return Ok(await _mediator.Send(new GetCountRequest(Request), cancellationToken));
+            var filtering = Request.ExtractFilteringRequest<ParcelFilter>();
+            var sorting = Request.ExtractSortingRequest();
```

Comment:
@ArneD shouldn't sorting also be a default value in the Count method?

Reply:
It is default if there is no header